boa_parser/src/parser/expression/update.rs

```diff
@@ -56,6 +56,19 @@ impl UpdateExpression {
     }
 }
 
+fn is_simple(expr: &Expression, position: Position, strict: bool) -> ParseResult<bool> {
```

Comment:
Could we add a link to the spec explaining what a "simple" target type is?

Reply:
I've added a link but it explains "simple" target type in terms of syntax-directed operation. My implementation is a function that takes a few arguments. Is that ok?

Reply:
I think that's OK, I would just use doc comments `///` instead of `//`, use `<` and `>` to enclose the link, and add a small explanation "this function checks if the target type is simple".

Reply:
Thanks for the suggestion. I made the changes.